.travis.yml

```diff
@@ -4,7 +4,7 @@ before_install:
    - sudo apt-get -y install libtool automake autoconf autoconf-archive
    - sudo apt-get -y install texlive-latex-base texlive-latex-recommended
    - sudo apt-get -y install texlive-latex-extra texlive-fonts-recommended texinfo
-   - sudo apt-get -y install python-pygments python-docutils python-sphinx
+   - sudo pip install pygments docutils sphinx
```

Comment:
docs changes are uninvited guest here, right?

Reply:
ah yes, sorry, will fix in a sec

Reply:
fixed in last push, I’m keeping the template rename since this breaks the build